improver/categorical/decision_tree.py

```diff
@@ -420,7 +418,8 @@ def create_condition_chain(self, test_conditions: Dict) -> List:
                 d_threshold_index = -1
                 extract_constraint = []
                 for item in diagnostic:
-                    if is_variable(item) or d_threshold is None:
+                    if is_variable(item):
+                        print(d_threshold)
```

Comment:
Unexpected print statement - you probably forgot to remove it.

Reply:
Removed